storage/innobase/trx/trx0sys.cc

```diff
@@ -1253,15 +1280,42 @@ ulint
 trx_sys_any_active_transactions(void)
 /*=================================*/
 {
-	ulint	total_trx = 0;
-
 	mutex_enter(&trx_sys->mutex);
 
-	total_trx = UT_LIST_GET_LEN(trx_sys->rw_trx_list)
-		  + UT_LIST_GET_LEN(trx_sys->mysql_trx_list);
+	ulint	total_trx = UT_LIST_GET_LEN(trx_sys->mysql_trx_list);
+
+	if (total_trx == 0) {
```

Comment:
The previous version of this function used to return
`UT_LIST_GET_LEN(trx_sys->rw_trx_list) + UT_LIST_GET_LEN(trx_sys->mysql_trx_list) - trx_sys->n_prepared_trx`
The new one, when `UT_LIST_GET_LEN(trx_sys->mysql_trx_list)  != 0` returns just `UT_LIST_GET_LEN(trx_sys->mysql_trx_list)`, ignoring `UT_LIST_GET_LEN(trx_sys->mysql_trx_list)` and `trx_sys->n_prepared_trx`
Is this OK?